parallel_cluster/README.md

```diff
@@ -31,9 +31,14 @@ $ ssm i-XXXXXXXXXXXXX
 ```
 
 ### Running the cluster
+> CLUSTER_NAME can only be alpha-numeric and must start with a letter
 
 ```shell
-$ ./bin/start-cluster.sh <CLUSTER_NAME>
+$ CLUSTER_TEMPLATE="tothill"  # or umccr_dev 
+$ ./bin/start_cluster.sh \
+  --cluster-name <CLUSTER_NAME> 
+  --cluster-template "${CLUSTER_TEMPLATE}"
+  --extra-parameters "{\"KeyName\": \"alexis-wfh-dev\"}"
```

Comment:
On the keyname discussion: please use and encourage SSM instead, we'll be better off if we don't have to track public SSH keys in-AMI or instance (plus we can log all SSM user activity on an S3 bucket and audit properly).

Reply:
If I don't place in KeyName in either the config or as extra-parameters, I get an error stating that this attribute is missing. How do we create the instance if we don't have a KeyName to begin with?

Reply:
Yes, I think you need an SSH key, but that does not have to be yours. It can be any that's available in the account and it should (generally) be ignored. I'd probably set a default one in the `config` and forget about it.

Reply:
Right and then install ssm into the AMI s.t we can run ssm inside?

Reply:
I suspect that Amazon Linux 2 (for which I'm basing those EC2 Image Builder components off of) already has SSM reqs installed?

Reply:
It should yes. You just need to make sure to include the permissions in the instance profile...
I think `AmazonSSMManagedInstanceCore` should do it.

Reply:
I added in AmazonSMMManagedInstanceCore as an additionalPolicy in the config and this enabled me to SSM into the instance.
I've also placed some commands in the base.yml image builder to ensure the latest version of SSM is installed on the image.